### PR TITLE
Cheerpx docs update review

### PR DESCRIPTION
**Pull request by alexp-sssup**

### CODE REVIEW

File: sites/cheerpx/src/content/docs/11-guides/File-System-support.md [hunk 1]
```diff
@@ -58,12 +58,47 @@ Create an IDBDevice using the `CheerpX.IDBDevice.create()` method:
...
+
+Example:
+
+```js
+const idbDevice = await CheerpX.IDBDevice.create("files");
+// Use CheerpX to write something to the device
+const outputBlob = await dataDevice.readFileAsBlob("/filename");
+```
+
+> [!note] > [!note] Note
```
Comment (GabrielaReyna) [inline sites/cheerpx/src/content/docs/11-guides/File-System-support.md:99]:
```suggestion
> [!note] Note
```

File: sites/cheerpx/src/content/docs/11-guides/File-System-support.md
Comment (GabrielaReyna) [inline sites/cheerpx/src/content/docs/11-guides/File-System-support.md:1]: There is another callout without title in this file

File: sites/cheerpx/src/content/docs/20-faq.md [hunk 2]
```diff
@@ -10,30 +10,42 @@ Currently, CheerpX doesn't directly support capturing stdout from running progra
...
 
-## Why can't CheerpX find files in my device?
-
-When using WebDevice, keep in mind:
+## Why can't CheerpX find files in my WebDevice backend?
 
-1. WebDevice uses absolute paths from the root of the mounted directory.
-2. Do not include the first slash `/` of the path.
-3. The path is relative to the root of the mounted WebDevice, not the current working directory.
+We know from experience that the interaction between mount points and `WebDevice` can be confusing for some users. The best solution to identify why a file can't be found is to use the "Network" tab to see the final URLs that CheerpX is ttying to access. With this information you should be able to fix the incorrect paths.
```
Comment (GabrielaReyna) [inline sites/cheerpx/src/content/docs/20-faq.md:48]:
```suggestion
We know from experience that the interaction between mount points and `WebDevice` can be confusing for some users. The best solution to identify why a file can't be found is to use the _**Network**_ tab to see the final URLs that CheerpX is trying to access. With this information you should be able to fix the incorrect paths.
```

File: sites/cheerpx/src/content/docs/20-faq.md [hunk 3]
```diff
@@ -45,26 +57,26 @@ Use browser's DevTools:
 4. Look for 404 (Not Found) errors.
 5. Check the full URL of these 404 requests to see the exact path CheerpX is trying to access.
 
-## Why can't I execute files directly from a DataDevice?
+## Can I use third-party origins with WebDevice?
```
Comment (GabrielaReyna) [inline sites/cheerpx/src/content/docs/20-faq.md:60]:
```suggestion
## Can I use third-party origins with `WebDevice`?
```

**Review by GabrielaReyna — CHANGES_REQUESTED** [review]
I have made some formatting suggestions. Please let me know if you are happy for me to take over from here and format other parts of the file.

**Comment by alexp-sssup** [comment]
Applied all your fixes. Thanks.

**Review by Honggeiry — CHANGES_REQUESTED** [review]
There's only a small error in the variable name, everything else looks good

**Pull request merged**